components/app/landing/LandingLatestProjects.vue

```diff
@@ -1,90 +1,78 @@
 <template>
   <LandingSectionContainer class="my-32">
-    <h2>{{ $t('landing.latest_projects.title') }} <span class="text-primary">{{ $t('landing.latest_projects.titleHighlight') }}</span></h2>
+    <h2>
+      {{ $t('landing.latest_projects.title') }}
+      <span class="text-primary">{{
+        $t('landing.latest_projects.titleHighlight')
+      }}</span>
+    </h2>
     <p>{{ $t('landing.latest_projects.text') }}</p>
-    <div class="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 2xl:grid-cols-5 gap-4 mt-10">
-      <div v-for="project in projects" :key="project.title">
-      <NuxtLink :prefetch="false" to="/project-overview">
-      <div class="shadow rounded p-4 pb-12 h-275px text-center">
-        <div class="mt-3">
-          <img class="inline-block h-16 w-16 rounded-full ring-2 ring-white"
-            :src="project.imageURL" :alt="project.title" />
-        </div>
-        <div class="mt-4">
-          <p class="text-primary p-line-head">{{project.title | truncate(14)}}</p>
-          <p class="small-text">{{project.category}}</p>
-        </div>
-        <div class="mt-4 flex place-content-center">
-          <p class="border small-text text-primary w-32">{{project.createdAt}}</p>
-        </div>
+    <div
+      class="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 2xl:grid-cols-5 gap-4 mt-10"
+    >
+      <div v-for="project in projects" :key="project._id">
```

Comment:
issue (blocking): same issue as in previous comments. I will not comment any further. Please update if this is meant to be the MongoDB ID